GitUI/UserControls/BlameControl.cs

```diff
@@ -109,22 +139,22 @@ private void BlameFile_MouseMove(object sender, MouseEventArgs e)
                 return;
             }
 
-            GitBlameHeader blameHeader = _blame.FindHeaderForCommitGuid(_blame.Lines[line].CommitGuid);
+            GitBlameCommit blameCommit = _blame.Lines[line].Commit;
 
-            if (blameHeader != _lastBlameHeader)
+            if (blameCommit != _lastBlameCommit)
             {
                 BlameCommitter.ClearHighlighting();
                 BlameFile.ClearHighlighting();
                 int startLine = -1;
                 int prevLine = -1;
                 for (int i = 0; i < _blame.Lines.Count; i++)
                 {
-                    if (_blame.Lines[i].CommitGuid == blameHeader.CommitGuid)
+                    if (ReferenceEquals(_blame.Lines[i].Commit, blameCommit))
                     {
                         if (prevLine != i - 1 && startLine != -1)
                         {
-                            BlameCommitter.HighlightLines(startLine, prevLine, Color.FromArgb(225, 225, 225));
-                            BlameFile.HighlightLines(startLine, prevLine, Color.FromArgb(225, 225, 225));
+                            BlameCommitter.HighlightLines(startLine, prevLine, Color.White);
+                            BlameFile.HighlightLines(startLine, prevLine, Color.White);
```

Comment:
We should be conscious of dark themes and implications of explicit colour definitions.
Perhaps this needs to be one of the `SystemColors` instead.

Reply:
Good idea. Done.